scripts/src/main/resources/setup.bat

```diff
@@ -4,6 +4,10 @@ if NOT "%DEVON_IDE_TRACE%"=="" echo on
 Set _fBGreen=[92m
 Set _RESET=[0m
 
+IF NOT %1.==_isNew. echo %PSModulePath% | findstr %USERPROFILE% >NUL
+IF NOT %1.==_isNew. IF %ERRORLEVEL% EQU 0 start "CMD window" "%~f0" _isNew & goto :EOF
+echo Running from cmd.exe...
+
```

Comment:
> So if I now got it correctly, this is the fix for #866 and the changes in `devon.bat` for the fix for #409. Next time it is better, if you create separate PRs for each issue but thanks for clarification.
> 
> This code could be written a bit more intuitive like this:
> 
> ```
> IF NOT %1.==_isNew. (
>   echo %PSModulePath% | findstr %USERPROFILE% >NUL
>   IF %ERRORLEVEL% EQU 0 start "CMD window" "%~f0" _isNew & goto :EOF
> )
> ```
> 
> Or wouldn't it be even much easier to just call `start "CMD window" scripts\devon.bat ide setup %*` instead and avoid this "hack" with the `_isNew` parameter?

The problem with starting command line the way you described would be how do you detect if the script is running in the powershell and not in the cmd.exe. If there is no detection involved, it would start the devon.bat script even if it is already running from the cmd.exe. The _isNew parameter is used to know if the current execution of the script is being performed in the new window or not. I am still unsure, but I think `%USERPROFILE%` is used for detection. I'll do a bit more testing and report about the results

Reply:
As of writing of this I have not found a better and more elegant solution than you have proposed already.

Reply:
For the record:
Simply run `cmd` in PowerShell and also directly via start-menu and do `echo %PSModulePath%`.
If you compare the two, you will notice that in `cmd` inside PowerShell you will have one entry more:
```
C:\Users\hohwille\Documents\WindowsPowerShell\Modules;
```

This is what you use to detect the difference. In `cmd` in PowerShell you then have to launch our `devon.bat` script again (by itself) via `start` command. To prevent causing an infinity loop, you then have to detect somehow that even though you initially have been started from PowerShell you are now inside an external `CMD` window.
You are doing this with the `_isNew` paramter from the workaround on the web.
An alternative would have been to remove the extra entry from `PSModulePath` (`%USERPROFILE%\Documents\WindowsPowerShell\Modules;`).
However, as this `_isNew` parameter also works I am fine with merging...